### PR TITLE
[Refactor] update TIN readme

### PR DESCRIPTION
**Pull request by cir7**
## Motivation
retrain tin  model and  update TIN readme

### CODE REVIEW

File: configs/recognition/tin/tin_imagenet-pretrained-r50_8xb6-1x1x8-40e_sthv1-rgb.py [hunk 1]
```diff
@@ -99,7 +99,7 @@
 optim_wrapper = dict(
     optimizer=dict(
         type='SGD',
-        lr=0.02,
+        lr=0.01,
```
Comment (hukkai) [inline configs/recognition/tin/tin_imagenet-pretrained-r50_8xb6-1x1x8-40e_sthv1-rgb.py:102]: why changing the learning rate?

File: configs/recognition/tin/tin_kinetics400-pretrained-tsm-r50_1x1x8-50e_kinetics400-rgb.py [hunk 2]
```diff
@@ -99,3 +99,4 @@
 # optimizer
 optim_wrapper = dict(
     constructor='TSMOptimWrapperConstructor', paramwise_cfg=dict(fc_lr5=True))
+load_from = 'https://download.openmmlab.com/mmaction/recognition/tsm/tsm_r50_1x1x8_50e_kinetics400_rgb/tsm_r50_1x1x8_50e_kinetics400_rgb_20200607-af7fb746.pth'  # noqa: E501
```
Comment (hukkai) [inline configs/recognition/tin/tin_kinetics400-pretrained-tsm-r50_1x1x8-50e_kinetics400-rgb.py:102]: is it better to use the `pretrained` option like https://github.com/open-mmlab/mmaction2/blob/439bfcedc55562adada98997713c5257149974f5/configs/recognition/slowonly/slowonly_imagenet-pretrained-r50-nl-embedded-gaussian_8xb16-4x16x1-steplr-150e_kinetics400-rgb.py#L6

**Review by hukkai — APPROVED** [review]

File: configs/recognition/tin/tin_imagenet-pretrained-r50_8xb6-1x1x8-40e_sthv1-rgb.py (hunk 1 above)
Comment (cir7) [reply configs/recognition/tin/tin_imagenet-pretrained-r50_8xb6-1x1x8-40e_sthv1-rgb.py:102]: @hukkai smaller lr gets a better result

File: configs/recognition/tin/tin_kinetics400-pretrained-tsm-r50_1x1x8-50e_kinetics400-rgb.py (hunk 2 above)
Comment (cir7) [reply configs/recognition/tin/tin_kinetics400-pretrained-tsm-r50_1x1x8-50e_kinetics400-rgb.py:102]: @hukkai ok, i will modify it

Comment (cir7) [reply configs/recognition/tin/tin_kinetics400-pretrained-tsm-r50_1x1x8-50e_kinetics400-rgb.py:102]: `pretrained` is used to load backbone ckpt, but `load form` is for the `recognizer`, and its ckpt has a prefix `backbone.`
another option is use `init_cfg`, which support remove prefix, but ResNetTIN pass the `init_weights` due to its weird initialization flow.
so we can only use `load_from`  here.
